lib/src/dart_code_viewer.dart

```diff
@@ -375,21 +389,28 @@ class DartCodeViewer extends StatelessWidget {
     );
   }
 
-  InlineSpan codifyString(String content, DartCodeViewerThemeData dartCodeViewerThemeData) {
-    List<TextSpan> textSpans = [];
-    final codeSpans = DartSyntaxPrehighlighter().format(content);
+  InlineSpan codifyString(
+    String content,
+    DartCodeViewerThemeData dartCodeViewerThemeData,
+  ) {
+    var textSpans = <TextSpan>[];
```

Comment:
This can be `final`: since `List` is mutable, changing the contents of the list does not change the list itself.

Reply:
Done